### PR TITLE
Fixed small issues in Appendix D

### PR DESCRIPTION
**Pull request by rsignell-usgs**
I tried to fix all the remaining issues in Appendix D, mostly fixing formatting of variables

### CODE REVIEW

File: appd.adoc [hunk 1]
```diff
@@ -171,7 +167,7 @@ z(n,k,j,i) = a(k)*ztop + b1(k)*zsurf1(n,j,i) + b2(k)*zsurf2(n,j,i)
               
 ----
 
-where **`z(n,k,j,i)`**is the height above the geoid (approximately mean sea level) at gridpoint **`(k,j,i)`**and time **`(n)`**, **`ztop`**is the height of the top of the model, and **`a(k)`**, **`b1(k)`**, and **`b2(k)`**are the dimensionless coordinates which define hybrid level **`k`**. **`zsurf1(n,j,i)`**and **`zsurf2(n,j,i)`**are respectively the large and small parts of the topography. See Shaer et al [<<sch02,SCH02>>]for details.
+where **`z(n,k,j,i)`**is the height above the geoid (approximately mean sea level) at gridpoint **`(k,j,i)`**and time **`(n)`**, **`ztop`**is the height of the top of the model, and **`a(k)`**, **`b1(k)`**, and **`b2(k)`**are the dimensionless coordinates which define hybrid level **`k`**. **`zsurf1(n,j,i)`**and **`zsurf2(n,j,i)`** are respectively the large and small parts of the topography. See Shaer et al [<<sch02,SCH02>>] for details.
```
Comment (rhattersley) [inline appd.adoc]: It'd be nice to have a space after `zsurf1` in the prose, just like you've done for `zsurf2`.

Comment (rhattersley) [inline appd.adoc]: That also applies to a few other snippets too: `z(n,k,j,i)`, `(k,j,i)`, `ztop`, and `b2`.

File: appd.adoc [hunk 2]
```diff
@@ -317,7 +313,7 @@ f(j,i)= 0.5*(z1+ z2) + 0.5*(z1-z2)* tanh(2*a/(z1-z2)*(depth(j,i)-href))
               
 ----
 
-where **`z(k,j,i)`**is height, positive upwards, relative to ocean datum (e.g. mean sea level) at gridpoint **`(k,j,i)`**, **`sigma(k)`**is the dimensionless coordinate at vertical gridpoint **`(k)`**for **`k &lt;= k_c`**, and **`depth(j,i)`**is the distance from ocean datum to sea floor (positive value) at horizontal gridpoint **`(j,i)`**. z1, z2, a, and hrefare constants.
+where **`z(k,j,i)`**is height, positive upwards, relative to ocean datum (e.g. mean sea level) at gridpoint **`(k,j,i)`**, **`sigma(k)`**is the dimensionless coordinate at vertical gridpoint **`(k)`**for **`k &lt;= k_c`**, and **`depth(j,i)`**is the distance from ocean datum to sea floor (positive value) at horizontal gridpoint **`(j,i)`**. **`z1`**, **`z2`**, **`a`**, and **`href`**are constants.
```
Comment (rhattersley) [inline appd.adoc]: Would you mind checking the other inline-code fragments in this paragraph and adding trailing spaces as appropriate?

**Comment by rsignell-usgs** [comment]
@rhattersley , @ocefpaf pointed out to me that the issue with these trailing spaces is systematic, and probably could be addressed more effectively by fixing the conversion script XSL.   Do you agree?

**Comment by rhattersley** [comment]
> probably could be addressed more effectively by fixing the conversion script XSL. Do you agree?

More effectively? Probably.

More quickly? Possibly not! The XSL is pretty gnarly and sadly I have very little time to spare. Perhaps a simpler search & replace on the current AsciiDoc files would give us an 80/20 (or better!) solution?

**Comment by rsignell-usgs** [comment]
okay, I have modified the rest (I think) of the trailing spaces by hand.

**Comment by rhattersley** [comment]
Thanks @rsignell-usgs. :+1:

I know it's not perfect, but I don't think that matters ... it's much better than it was! Let's get this merged...

**Comment by rhattersley** [comment]
NB. The Travis-CI build for the merge (which will update the built HTML) is https://travis-ci.org/cf-metadata/cf-conventions/builds/68023496

**Comment by ocefpaf** [comment]
I guess you still need to add a new TOKEN, right?

https://travis-ci.org/cf-metadata/cf-conventions/builds/68023496#L127

That kind of failure does not trigger a travis faluire.

**Comment by rhattersley** [comment]
> I guess you still need to add a new TOKEN, right?

Yes - that's done now. Thanks again for the token-leak warning.

> That kind of failure does not trigger a travis faluire.

Shame. :unamused:

**Pull request merged**